### PR TITLE
Pass request to serializer

### PR DESCRIPTION
**Pull request by Salomari1987**
### Description

TASK: [Internal server error when un-ernrolling from a course then enroll again](https://edraak.atlassian.net/browse/EMS-126)

### CODE REVIEW

File: common/djangoapps/enrollment/views.py [hunk 1]
```diff
@@ -524,17 +524,18 @@ def post(self, request):
                     "message": u"No course '{course_id}' found for enrollment".format(course_id=course_id)
                 }
             )
-
```
Comment (saadaltabari) [inline common/djangoapps/enrollment/views.py]: can you just remove these line deletion and adding changes

File: common/djangoapps/enrollment/views.py [hunk 2]
```diff
@@ -524,17 +524,18 @@ def post(self, request):
...
         mode = request.data.get('mode')
 
         has_api_key_permissions = self.has_api_key_permissions(request)
 
         # Check that the user specified is either the same user, or this is a server-to-server request.
         if not username:
             username = request.user.username
+
```
Comment (saadaltabari) [inline common/djangoapps/enrollment/views.py]: can you just remove these line deletion and adding changes

File: common/djangoapps/enrollment/views.py [hunk 3]
```diff
@@ -524,17 +524,18 @@ def post(self, request):
...
         if username != request.user.username and not has_api_key_permissions:
             # Return a 404 instead of a 403 (Unauthorized). If one user is looking up
             # other users, do not let them deduce the existence of an enrollment.
+
```
Comment (saadaltabari) [inline common/djangoapps/enrollment/views.py]: can you just remove these line deletion and adding changes

File: common/djangoapps/enrollment/views.py [hunk 4]
```diff
@@ -610,6 +614,7 @@ def post(self, request):
                     is_active=is_active,
                     enrollment_attributes=enrollment_attributes
                 )
+
```
Comment (saadaltabari) [inline common/djangoapps/enrollment/views.py]: can you just remove these line deletion and adding changes

File: common/djangoapps/enrollment/views.py [hunk 5]
```diff
@@ -586,6 +589,7 @@ def post(self, request):
                 ]
                 missing_attrs = set(REQUIRED_ATTRIBUTES.get(mode, [])) - set(actual_attrs)
             if has_api_key_permissions and (mode_changed or active_changed):
+
```
Comment (saadaltabari) [inline common/djangoapps/enrollment/views.py]: can you just remove these line deletion and adding changes

**Comment by halawa** [comment]
@Salomari1987 basha, may you please open a PR of this fix into master? Thanks so much :)

**Pull request merged**